posts/2023/unstructured-grid-collab-1.md

```diff
@@ -0,0 +1,106 @@
+
+**Topic:** Project [Raijin](https://raijin.ucar.edu/) & [UXarray](https://uxarray.readthedocs.io/)
+
+*What did your group do?*
+
+The UXarray devs had a chance to discuss use cases and get feedback from the community members on UXarray. We branched into subgroups for different work ideas and coded with our users. We noticed a significant bug and fixed that throughout the end of the day.
+
+*Was it a fruitful experience?*
+
+Very much!
+
+*What worked for your group and what didn't work?*
+
+Sitting together with the users and listening to their expectations was a great opportunity! Since we had multiple UXarray folks as facilitators, things worked quite well for us. That wouldn't happen if we didn't have that many helpers (as a side note for the future)
+
+*Would it be useful to redo the  topic again?  What would you do differently the second time around?*
+
+Yes, after some time. We'd add one or two work idea of our own group as well as those came from the community. There were bugs related to
+1. UXarray's MPAS data ingest (attributes not being parsed) with the dual mesh only (see PRs [#274](https://github.com/UXARRAY/uxarray/pull/274) and [#275](https://github.com/UXARRAY/uxarray/pull/275)), and
+2. face area calculations (see PR [#283](https://github.com/UXARRAY/uxarray/pull/283))
+
+
+#### Falko Judt (MMM):
+
+**Topic** Extract Tropical Cyclone Vitals from MPAS
+
+*What did your group do?*
+
+ Free discussion. Mostly scientist asking questions to the GeoCAT/UXarray experts. We didn't write any code because nobody had a project ready.
+
+*Was it a fruitful experience?*
+
+ Yes.
+
+*What worked for your group and what didn't work?*
+
+Discussion was great. Also just meeting some people face to face was a plus.
+
+*Would it be useful to redo the  topic again?  What would you do differently the second time around?*
+
+Yes, maybe in 1/2 year or so? Could tell people to have some project ready to get into some hands-on coding.
+
+
+#### Katie Dagon (CGD):
+
+**Topic**: Visualizing & Analyzing CAM-SE output in Python
+
+*What did your group do? Write code? discussions?*
+
+We started with open discussion and sharing our experiences with the overall topic (CAM-SE analysis/visualization).
+We then went through [a notebook](https://github.com/katiedagon/CAM-SE_analysis/blob/main/notebooks/CAM-SE.ipynb) I had prepared with various examples of remapping to visualize CAM-SE unstructured output.
+I did a little live debugging, then [Deepak was kind enough](https://github.com/katiedagon/CAM-SE_analysis/pull/1) to help debug the xemsf code!
+We ended with some additional discussion and had a few more participants log on after their first breakout sessions ended early.
+In total I believe we had 9 active participants (myself included) and went for at least ~1.5 hours.
+
+*Was it a fruitful experience?*
+
+Yes, very. A lot of good information and helpful tips/tools were exchanged. I can list some of these out for the blog post if that would be helpful.
```

Comment:
You can remove the second sentence here, and add some takeaways/resources:
- Users need to consider different tools for different purposes (e.g., visualization, analysis, looking at extremes).
- Bilinear interpolation will change information but is generally acceptable when going from coarse to fine scale. Consider conservative remapping when going from fine to coarse scale, and nearest-neighbor method for preserving extremes.
- If you have access to NCAR HPC, CESM grid files are located at `/glade/p/cesmdata/cseg/inputdata/share/`
- [ncremap](https://github.com/nco/nco/blob/master/data/ncremap) via NCO tools is still a solid resource for remapping, though we are interested in Python-based alternatives.
- [VisIt](https://visit-dav.github.io/visit-website/index.html) and [ncvis](https://github.com/SEATStandards/ncvis) are useful for quick visualization of unstructured grids.
- [TempestRemap](https://github.com/ClimateGlobalChange/tempestremap) can be used for native spectral element regridding.
- Ongoing UXarray development would hopefully make this process more seamless, so the user does not have to interact with the native grid. This is still in process.